_includes/selector.html

```diff
@@ -644,25 +652,32 @@
             },
             getDockerNotes() {
                 var notes = [];
-                if (this.active_docker_cuda_ver.startsWith("12") && this.active_release === "Stable") {
-                    var pkgs_html = this.rapids_meta_pkgs.map(pkg => "<code>" + pkg + "</code>").join(", ");
-                    notes = [...notes]
-                } else {
-                    var pkgs_html = this.rapids_meta_pkgs.map(pkg => "<code>" + pkg + "</code>").join(", ");
+                var pkgs_to_show = this.rapids_meta_pkgs;
+
+                // Hide cuSpatial and cuProj for Nightly
+                if (this.active_release === "Nightly") {
+                    pkgs_to_show = pkgs_to_show.filter(pkg => pkg !== "cuSpatial" && pkg !== "cuProj");
                 }
 
+                var pkgs_html = pkgs_to_show.map(pkg => "<code>" + pkg + "</code>").join(", ");
                 notes = [...notes, "The selected image contains the following packages:<br>" + pkgs_html];
                 return notes.map(note => this.note_prefix + " " + note);
-
             },
             getCondaNotes() {
                 var notes = [];
                 if (this.active_conda_cuda_ver.startsWith("11")) {
                     notes = [...notes, "RAPIDS on CUDA 11 doesn't support <code>channel_priority: strict</code>; use <code>channel_priority: flexible</code> instead"];
                 }
 
+                var pkgs_to_show = this.rapids_meta_pkgs;
+
+                // Hide cuSpatial and cuProj for Nightly
+                if (this.active_release === "Nightly") {
+                    pkgs_to_show = pkgs_to_show.filter(pkg => pkg !== "cuSpatial" && pkg !== "cuProj");
```

Comment:
If you select stable, Choose specific packages, and select cuspatial. Then switch to Nightly, the cuspatial button is disabled, but the install script will still include the packages.

![image](https://github.com/user-attachments/assets/09b22020-3038-40e6-a9f2-9bc92bd8f518)

Reply:
Ah, great. I'll try to fix.

Reply:
Fixed in af79499. Merging.